### ISSUE TITLE
Collection items duplicated every run

### ISSUE THREAD

**Issue by jahmad** [jellyfin/jellyfin-plugin-tmdbboxsets#12]
Using: jellyfin 10.5.0 linuxserver docker, plugin 5.0.0.0

Everytime this plugin run its scans, collection items get appended to collection.xml, even if they are already there.

this is an example of collection.xml after 11 runs:

```
<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<Item>
  <ContentRating>PG</ContentRating>
  <Added>3/14/2020 12:59:54 AM</Added>
  <LockData>false</LockData>
  <LocalTitle>The Incredibles Collection</LocalTitle>
  <DisplayOrder>PremiereDate</DisplayOrder>
  <TmdbId>468222</TmdbId>
  <CollectionItems>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/The Incredibles (2004)/The.Incredibles.2004.PROPER.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
    <CollectionItem>
      <Path>/data/media/Movies/Animation/Incredibles 2 (2018)/Incredibles.2.2018.1080p.BluRay.H264.AAC-RARBG.mp4</Path>
    </CollectionItem>
  </CollectionItems>
</Item>%
```

**Comment by mcarlton00** [comment]
This was an error in the 10.5.x release cycle where collections didn't work properly.  It's been fixed in 10.6.x.

https://github.com/jellyfin/jellyfin/issues/2565

**Comment by jahmad** [comment]
yes, no longer an issue in 10.6
